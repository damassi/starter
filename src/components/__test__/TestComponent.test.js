import React from 'react'
import TestComponent from 'components/TestComponent'
import renderer from 'react-test-renderer'

describe('components/TestComponent.js', () => {
  it('does a snapshot', () => {
    const tree = renderer.create(<TestComponent />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
