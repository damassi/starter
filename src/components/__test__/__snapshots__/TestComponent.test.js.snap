exports[`components/TestComponent.js does a snapshot 1`] = `
<div>
  Hello!
</div>
`;
